client/my-sites/post-type-list/index.jsx

```diff
@@ -46,6 +46,7 @@ class PostTypeList extends Component {
 		super( ...arguments );
 
 		this.renderPostRow = this.renderPostRow.bind( this );
+		this.cellRendererWrapper = this.cellRendererWrapper.bind( this );
```

Comment:
Do we need to bind here? Just looking at some of the other affected files. If we assume that the method is only ever called as part of a render (not a callback), it's probably safe to omit the bind. Maybe we were just being safe with `bind` as some sort of guarantee that `this` is available.